g4f/Provider/base_provider.py

```diff
@@ -108,14 +118,16 @@ def create_async_generator(
         raise NotImplementedError()
 
 
-def create_event_loop():
+def create_event_loop() -> SelectorEventLoop:
     # Don't create a new loop in a running loop
-    if asyncio.events._get_running_loop() is not None:
-        raise RuntimeError(
-            'Use "create_async" instead of "create" function in a async loop.')
-
-    # Force use selector event loop on windows
-    return asyncio.SelectorEventLoop()
+    try:
```

Comment:
Probably my bad, I wanted to avoid using undocumented python functions.
If you want to make this work as intended, you probably have to do something like:
`    try:
        asyncio.get_running_loop()
    except RuntimeError:
        return SelectorEventLoop()

    raise RuntimeError(
            'Use "create_async" instead of "create" function in a async loop.')
`

Reply:
get_running_loop() throws an exception if there is no event loop.
Therefore you raise your exception only if the try: statement goes smoothly

Reply:
I changed it

Reply:
Looks good to me then